source/production/F0.Minesweeper.Logic/F0.Minesweeper.Logic.csproj

```diff
@@ -12,6 +12,8 @@
 		<AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleToAttribute">
 			<_Parameter1>F0.Minesweeper.Logic.Tests</_Parameter1>
 		</AssemblyAttribute>
+		<AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleToAttribute">
+			<_Parameter1>DynamicProxyGenAssembly2</_Parameter1>
+		</AssemblyAttribute>
```

Comment:
This was necessary because Moq needs to access internal classes/types:
Error:
```
System.ArgumentException : Cannot set up IMinelayer.PlaceMines because it is not accessible to the proxy generator used by Moq:
    Can not create proxy for method System.Collections.Generic.IEnumerable`1[F0.Minesweeper.Logic.Abstractions.Location] PlaceMines(System.Collections.Generic.IEnumerable`1[F0.Minesweeper.Logic.Abstractions.Location], UInt32, F0.Minesweeper.Logic.Abstractions.Location) because it or its declaring type is not accessible. Make it public, or internal and mark your assembly with [assembly: InternalsVisibleTo("DynamicProxyGenAssembly2")] attribute, because assembly F0.Minesweeper.Logic is not strong-named.
  Stack Trace: 
    Guard.IsVisibleToProxyFactory(MethodInfo method) line 115
    InvocationShape.ctor(LambdaExpression expression, MethodInfo method, IReadOnlyList`1 arguments, Boolean exactGenericTypeArguments, Boolean skipMatcherInitialization, Boolean allowNonOverridable) line 83
    ExpressionExtensions.<Split>g__Split|5_0(Expression e, Expression& r, InvocationShape& p, Boolean assignment, Boolean allowNonOverridableLastProperty) line 218
    ExpressionExtensions.Split(LambdaExpression expression, Boolean allowNonOverridableLastProperty) line 148
    Mock.SetupRecursive[TSetup](Mock mock, LambdaExpression expression, Func`4 setupLast, Boolean allowNonOverridableLastProperty) line 618
    Mock.Setup(Mock mock, LambdaExpression expression, Condition condition) line 522
    Mock`1.Setup[TResult](Expression`1 expression) line 468
    MinefieldTest.Uncover_ForTheFirstTime_PlacesMinesOnlyOnce() line 129
```